features/tabelwaarden.feature

```diff
@@ -0,0 +1,90 @@
+Functionaliteit: Als developer wil ik waardetabellen gebruiken in plaats van enumeraties
+  zodat ik mijn code niet hoef aan te passen als een enumeratielijst wordt aangepast die ik in mijn functionaliteit niet gebruik.
+
+  Wanneer in de registratie specifieke waarden in een enumeratie voorkomen, worden deze waarden in de API doorgegeven middels een waardetabel.
+  Wanneer in de registratie een omschrijving gekoppeld is aan een specifieke enumeratiewaarde, wordt deze omschrijving raadpleegbaar gemaakt middels de waardetabellen API.
+
+
+  Voor de enumeratie AanduidingBijzonderNederlanderschap wordt de tabel met tabelidentificatie "aanduidingbijzondernederlanderschap" opgenomen.
+  In de omschrijving wordt de volgende tekst opgenomen : "Geeft aan dat de persoon behandeld wordt als Nederlander, of dat door de rechter is vastgesteld dat de persoon niet de Nederlandse nationaliteit bezit."
+  In de tabel worden de volgende waarden opgenomen:
+    | code                                             | omschrijving                     |
+    | ------------------------------------------------ | -------------------------------- |
+    | B                                                | Behandeld als nederlander        |
```

Comment:
we hebben destijds bij enumeraties gekozen om betekenisvolle waarden te definiëren. In dit voorstel laat je dat los. Dat heeft tegelijk als voordeel en nadeel dat de link met LO GBA direct duidelijk is (maar dus ook light coupling).
Maar dat betekent m.i. ook dat wanneer een waarde gebruikt wordt in code, hier een betekenisloze code komt. Dat maakt de code minder leesbaar en de kans op fouten groter. @MelvLee hoe zie jij dit?

Waarom niet:
| BEHANDELD_ALS_NEDERLANDER | Behandeld als Nederlander |

Reply:
Nog een nadeel van het gebruik van een betekenisvolle code is dat als de waarde bij de code een fout bevat de code zeer waarschijnlijk dezelfde fout zal bevatten. De waarde kan dan gecorrigeerd worden, de code corrigeren is lastiger.

Stel dus dat in de tabel 'redenopschortingbijhouding' in eerste instantie de foutieve waarde 'imigratie' was gebruikt dan was daarvan de code 'IMIGRATIE' geworden. De waarde kan dan eenvoudig gewijzigd worden in 'emigratie' maar de code niet eenvoudig in 'EMIGRATIE'.